models/classes/class.QtiTestConverter.php

```diff
@@ -286,7 +287,12 @@ private function arrayToComponent(array $testArray, QtiComponent $parent = null,
                             $this->arrayToComponent($value, $component, true);
                         } else {
                             $assignableValue = $this->componentValue($value, $class);
-                            if (! is_null($assignableValue)) {
+
+                            if ($assignableValue !== null) {
+                                if (is_string($assignableValue) && $key === 'content') {
+                                    $assignableValue = QtiTestSanitizer::sanitizeContent($assignableValue);
```

Comment:
can we ask for QtiTestSanitizer service from the service locator as we don't want to add more static calls ?

Reply:
I find it a bit weird to create a new class with just a static method to proxy another static method (with `htmlspecialchars_decode`). Can we call `Display::sanitizeXssHtml` directy here?
Static calls are not good but better than dependencies to service locator imho.